### DISCUSSION TITLE
Support for macOS Monterey

### DISCUSSION THREAD

**Discussion by infxmousrogue** [noah-nuebling/mac-mouse-fix#143] [Questions] [answered]
When is support for macOS Monterey coming? Currently I have not been able to get the app working on my macbook.
Thanks in advance!

**Comment by noah-nuebling** [comment, marked as answer]
Hey @infxmousrogue, sorry for the late reply. I'm marking this as answered assuming that the problem has been resolved for you. Please open an issue if you still have problems. Thanks!
